pod/locale/fr/LC_MESSAGES/django.po

```diff
@@ -7526,185 +7537,202 @@ msgstr ""
 "La vidéo \"%(content_title)s\" a été encodée au format web, et est "
 "maintenant disponible sur %(site_title)s."
 
-#: pod/video/views.py:343 pod/video/views.py:383
+#: pod/video/views.py:349 pod/video/views.py:389
 msgid "You cannot edit this channel."
 msgstr "Vous ne pouvez éditer cette chaîne."
 
-#: pod/video/views.py:890
+#: pod/video/views.py:896
 msgid "You cannot watch this video."
 msgstr "Vous ne pouvez pas voir cette vidéo."
 
-#: pod/video/views.py:924
+#: pod/video/views.py:930
 msgid "You cannot edit this video."
 msgstr "Vous ne pouvez pas éditer cette vidéo."
 
-#: pod/video/views.py:992
+#: pod/video/views.py:998
 msgid "You cannot delete this video."
 msgstr "Vous ne pouvez pas supprimer cette vidéo."
 
-#: pod/video/views.py:997
+#: pod/video/views.py:1003
 msgid "You cannot delete a video that is being encoded."
 msgstr "Vous ne pouvez pas supprimer une vidéo en cours d’encodage."
 
-#: pod/video/views.py:1007
+#: pod/video/views.py:1013
 msgid "The video has been deleted."
 msgstr "La vidéo a été supprimée."
 
-#: pod/video/views.py:1015
+#: pod/video/views.py:1021
 #, python-format
 msgid "Deleting the video \"%(vtitle)s\""
 msgstr "Supprimer la vidéo « %(vtitle)s »"
 
-#: pod/video/views.py:1132
+#: pod/video/views.py:1038
+msgid "Transcription not enable on this platform."
+msgstr ""
```

Comment:
Pourquoi ne pas traduire cette phrase ?

Reply:
… qui devrait d’ailleurs être « not enabled » ;)